README.md

```diff
@@ -12,24 +198,177 @@ command in the console. At the bottom, there will be information similar to the
+The other way of having dependencies in Ansible is by using the meta folder and main.yml inside the meta folder. Any tasks in meta/main.yml are run before the task/main.yml. An example of meta/main.yml:
 
-Options for docker_deploy playbook are [here](ansible/inventory/local/group_vars/docker_deploy.yml)
+```bash
+dependencies:
+  - { role: installation/aws-cli }
+```
 
-## How to run teleop_deploy playbook ##
+# Playbooks and possible command line arguments #
+
+## Playbook creation ##
+
+Create your playbook in ansible/playbooks folder. It has be a .yml file with no hyphens (underscores are allowed).
+
+You can read more about playbooks [here](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html)
+
+It has to have a similar structure to this:
+
+```bash
+---
+
+- name: Install Python 3
+  import_playbook: ./install_python3.yml
+
+- name: Install product Docker container and icons
+  hosts: docker_deploy
+  pre_tasks:
+
+    - name: No product is defined
+      when: product != 'hand_e' and product != 'hand_h'
+      meta: end_play
+
+    - name: check if customer_key is provided and not false
+      when: customer_key is defined and customer_key|bool
+      set_fact:
+        use_aws: true
+
+    - name: check if cyberglove branch is provided
+      when: cyberglove is defined and cyberglove|bool
+      set_fact:
+        use_cyberglove: true
+
+  roles:
+    - {role: installation/docker}
+    - {role: installation/nvidia-docker, when: nvidia_docker | int != 0}
+    - {role: products/hand-h/deploy, when: product == 'hand_h'}
+    - {role: products/hand-e/deploy, when: product == 'hand_e'}
+    - {role: docker/aws, when: use_aws|bool}
+```
+Key points:
+
+Always start by having:
+
+```bash
+- name: Install Python 3
+  import_playbook: ./install_python3.yml
+```
+This is to ensure Python3 is installed. Aurora uses Python3.
+
+Then you need to have one or more of task sections (or pre-task sections, which are executed before tasks) and an optional role section. Another example of a task section in a playbook (without role section):
+
+```bash
+- name: Check which hosts are available for teleop system Install
+  hosts: all
+  gather_facts: no
+  tasks:
+    - name: ping all the machines
+      ping:
+  become: yes
+```
+Some task sections specify "hosts", which tells Ansible which hosts to limit the execution to. You can read more about hosts in playbooks [here](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html#hosts-and-users)
+
+An example of a role section:
+
+```bash
+  roles:
+    - {role: installation/docker}
+    - {role: installation/nvidia-docker, when: nvidia_docker | int != 0}
+    - {role: products/hand-h/deploy, when: product == 'hand_h'}
+    - {role: products/hand-e/deploy, when: product == 'hand_e'}
+    - {role: docker/aws, when: use_aws|bool}
+```
+
+## teleop_deploy ##
+
+For deploying teleop software on multiple machines (server, control-machine, client, windows-machine)
+
+How to run:
```

Comment:
The only thing I didnt like is that now we are missing the quick guide that we have clearly described on top. 
If someones wants to quickly run aurora for docker deployment or teleop he will have to go through the document to find relevant sections. 
Maybe if we can have a very short section on top to show how to run it? and reference it to the proper section for more info

Reply:
Done